CHANGES.md

```diff
@@ -0,0 +1,13 @@
+Elgg Countries Plugin - CHANGE LOG
```

Comment:
My grunt script automatically creates a changelogs from commit messages. Use angular style commit messages for each feature and they will appear in the changelog